etc/sib-config/sib-config.yml

```diff
@@ -59,6 +59,32 @@ sib_networks:
   - "{{ sib_network_internal }}"
   - "{{ sib_network_demo_vlan }}"
   - "{{ sib_network_demo_vxlan }}"
+  - "{{ sib_network_public }}"
+
+# Public network name.
+sib_network_public_name: "public"
+
+# Public network.
+sib_network_public:
+  name: "{{ sib_network_public_name }}"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 79
+  shared: True
+  # Subnet configuration.
+  subnets:
+    - "{{ sib_subnet_public }}"
+
+# Public subnet.
+sib_subnet_public:
+  name: "{{ sib_network_public_name }}"
+  cidr: "10.12.9.0/24"
+  gateway_ip: "10.12.9.1"
+  allocation_pool_start: "10.12.9.10"
+  allocation_pool_end: "10.12.9.127"
+  # host_routes:
+  #   - destination: "10.4.0.0/16"
+  #     nexthop: "10.60.253.255"
 
 # SIB internal network name.
 sib_network_internal_name: "internal"
```

Comment:
Do we need this defining for Ironic?  The subnet pool seems a little odd and all.

Reply:
Seems we need this for routing from the provisioning network to the internal